From 3b3524cd9878e356f88075e570fa029b0790eca2 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Dan=20Hor=C3=A1k?= <dan@danny.cz>
Date: Fri, 28 May 2010 23:09:43 +0200
Subject: [PATCH] switch to CC variable for compiler

Don't use a hardcoded cross-gcc, switch to using CC variable instead.
---
 tools/env/Makefile |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/tools/env/Makefile b/tools/env/Makefile
index 2df631e..570a956 100644
--- a/tools/env/Makefile
+++ b/tools/env/Makefile
@@ -35,7 +35,7 @@ endif
 all:	$(obj)fw_printenv
 
 $(obj)fw_printenv:	$(SRCS) $(HEADERS)
-	$(CROSS_COMPILE)gcc $(CPPFLAGS) $(SRCS) -o $(obj)fw_printenv
+	$(CC) $(CPPFLAGS) $(SRCS) -o $(obj)fw_printenv
 
 clean:
 	rm -f $(obj)fw_printenv $(obj)crc32.c
-- 
1.6.6.1

